dll/win32/advapi32/service/sctrl.c

```diff
@@ -79,10 +79,10 @@ RPC_SERVICE_STATUS_HANDLE_unbind(RPC_SERVICE_STATUS_HANDLE hServiceStatus,
 }
 
 
-static RPC_STATUS
+static DWORD
 ScCreateStatusBinding(VOID)
 {
-    LPWSTR pszStringBinding;
+    RPC_WSTR pszStringBinding;
```

Comment:
Question: What's the benefit of using RPC_WSTR here?

Reply:
In principle nothing, but it's for consistency here: it's the type for "wide" (utf16) strings that are going to be sent over through RPC.